hooks/environment

```diff
@@ -32,15 +32,15 @@ case "${BUILDKITE_AGENT_META_DATA_CONFIG}" in
         echo "--- cpu env configuration"
         module load singularity/3.5.2
         module load python3/${PYTHON_VERSION:=3.8.5}
-        module load julia/${JULIA_VERSION:=1.7.1}
+        module load julia/${JULIA_VERSION:=1.8.0}
```

Comment:
this is just the default if it isn't otherwise set. I'm not sure we want to change it